### PR TITLE
[HYDRATOR-942] [HYDRATOR-945] minor enhancements

### PR DESCRIPTION
**Pull request by romy-khetan**

### CODE REVIEW

File: core-plugins/src/main/java/co/cask/hydrator/plugin/batch/source/ExcelInputReader.java [hunk 1]
```diff
@@ -419,7 +419,8 @@ private void getOutputSchema() {
 
     @Name("sheetValue")
     @Description("Specifies the value corresponding to 'sheet' input. Can be either sheet name or sheet no; " +
-      "for example: 'Sheet1' or '1' in case user selects 'Sheet Name' or 'Sheet Number' as 'sheet' input respectively.")
+      "for example: 'Sheet1' or '0' in case user selects 'Sheet Name' or 'Sheet Number' as 'sheet' input " +
```
Comment (abhinavbansal246) [inline core-plugins/src/main/java/co/cask/hydrator/plugin/batch/source/ExcelInputReader.java]: 'S' in Sheet to keep it in the same format as Sheet number

**Comment by abhinavbansal246** [comment]
LGTM

File: core-plugins/docs/Excel-batchsource.md [hunk 2]
```diff
@@ -34,12 +34,12 @@ a new table or existing one. (Macro-enabled)
...
-**sheet:** Specifies whether sheet has to be processed by sheet name or sheet no.
+**sheet:** Specifies whether sheet has to be processed by sheet name or sheet number.
 
 **sheetValue:** Specifies the value corresponding to 'sheet' input. Value can be either actual
-sheet name or sheet no.
-for example: 'Sheet1' or '1' in case user selects 'Sheet Name' or 'Sheet Number' as 'sheet'
-input respectively. (Macro-enabled)
+sheet name or sheet number.
+for example: 'Sheet1' or '0' in case user selects 'Sheet Name' or 'Sheet Number' as 'sheet'
+input respectively. Sheet number starts with 0. (Macro-enabled)
```
Comment (abhinavbansal246) [inline core-plugins/docs/Excel-batchsource.md:42]: Sheet number is still starting from 0?

Comment (romy-khetan) [reply core-plugins/docs/Excel-batchsource.md:42]: In Excel POI api, sheet number starts at 0. Therefore the implementation is done based on the sheet number starting with 0.

Comment (abhinavbansal246) [reply core-plugins/docs/Excel-batchsource.md:42]: @russorat Do you have any suggestions regarding this?

**Comment by abhinavbansal246** [comment]
Discussed with @russorat 
LGTM

**Comment by abhinavbansal246** [comment]
rebase and squash commits

**Comment by romy-khetan** [comment]
Squashed all the commits.

**Comment by abhinavbansal246** [comment]
http://builds.cask.co/browse/HYP-BAD-21

**Pull request merged**